src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -2197,6 +2198,12 @@ public boolean isBrowserReusable(SubversionSCM x, SubversionSCM y) {
             return true;
         }
 
+        @Nonnull
+        @Override
+        public Permission getRequiredGlobalConfigPagePermission() {
+            return Jenkins.MANAGE;
```

Comment:
Changing this permission check is enough? there is no other permission check in the submitting, or in the jelly template?

Reply:
> there is no other permission check in the submitting

This is submitted as part of the global configuration and it is already [checking for Manage](https://github.com/jenkinsci/jenkins/blob/78ea4b6b558362356f9a59b627fdc78ad21e4569/core/src/main/java/jenkins/model/Jenkins.java#L3803)

> or in the jelly template?

No, this is a `global.jelly`, already handled by core.